src/flint/test/test_all.py

```diff
@@ -2981,7 +3024,7 @@ def quick_poly():
                 (0, 1): 6
             })
 
-        for T in [int, S, flint.fmpz, lambda x: P(x, ctx=ctx)]:
+        for T in [int, S, int, lambda x: P(x, ctx=ctx)]:
```

Comment:
This is checking `int` twice. Should it not check `fmpz`?

Reply:
It certainly should be, whoops